src/Symfony/Component/Form/Extension/Core/DataTransformer/IntegerToLocalizedStringTransformer.php

```diff
@@ -24,19 +24,23 @@ class IntegerToLocalizedStringTransformer extends NumberToLocalizedStringTransfo
     /**
      * Constructs a transformer.
      *
-     * @param bool $grouping     Whether thousands should be grouped
-     * @param int  $roundingMode One of the ROUND_ constants in this class
+     * @param bool        $grouping     Whether thousands should be grouped
+     * @param int         $roundingMode One of the ROUND_ constants in this class
+     * @param string|null $locale       locale used for transforming
      */
-    public function __construct($grouping = false, $roundingMode = self::ROUND_DOWN)
+    public function __construct($grouping = false, $roundingMode = self::ROUND_DOWN, $locale = null)
     {
         if (\is_int($grouping) || \is_bool($roundingMode) || 2 < \func_num_args()) {
             @trigger_error(sprintf('Passing a precision as the first value to %s::__construct() is deprecated since Symfony 4.2 and support for it will be dropped in 5.0.', __CLASS__), \E_USER_DEPRECATED);
 
             $grouping = $roundingMode;
             $roundingMode = 2 < \func_num_args() ? func_get_arg(2) : self::ROUND_DOWN;
         }
```

Comment:
I think, this deprecation layer needs some adjustments now that you've added a third parameter.

Reply:
What's the previous signature of __construct? I don't see it explicitly written somewhere.

Reply:
Found it in branch 4.1.